capstone/src/main/webapp/script.js

```diff
@@ -126,4 +126,33 @@ function getRegMessage() {
   }
   const node = document.importNode(template.content, true);
   document.body.appendChild(node);  
+}
+
+function showEdit() {
+  document.getElementById('description').contentEditable = 'true';
+  document.getElementById('website').contentEditable = 'true';
+  document.getElementById('officers-list').contentEditable = 'true';
```

Comment:
nit: I think you should be able to define contenteditable="true" in the beginning portion of div tags directly in about-us.html. So if you think this will be cleaner, you could change the p tags to div tags for description and website. Not sure if it works for ul tags though

Reply:
To clarify, I wanted to define them in this method b/c I didn't want the tags to be editable from the beginning. until they click the "edit" button which calls this method to make them editable

Reply:
Ah I see, sorry about that :)